cmd/utils.go

```diff
@@ -369,3 +370,30 @@ func isNetworkOrHostDown(err error) bool {
 	}
 	return false
 }
+
+// getMacAddr returns all the available mac addresses
+// on the system
+func getMacAddr() (addr []string) {
+	interfaces, err := net.Interfaces()
+	if err == nil {
+		for _, i := range interfaces {
+			if i.Flags&net.FlagUp != 0 && bytes.Compare(i.HardwareAddr, nil) != 0 {
+				// Don't use random as we have a real address
+				addr = append(addr, i.HardwareAddr.String())
+			}
+		}
+	}
+	return
+}
+
+func totalSpace(device string, stats []disk.PartitionStat) uint64 {
```

Comment:
Can you add more comments which describes this function ?

Reply:
done